src/Exception/MaintenanceModeActiveHttpException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\ServiceUnavailableHttpException;
+
+/**
+ * This exception is thrown when maintenance mode is active.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class MaintenanceModeActiveHttpException extends ServiceUnavailableHttpException
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(
+        // FIXME: should we initialize this to a sane default?
```

Comment:
I'm voting for setting a default of one hour. The RFC 2616 section "10.5.4 503 Service Unavailable" states

> If no Retry-After is given, the client SHOULD handle the response as it would for a 500 response. 

which again means 

> The server encountered an unexpected condition which prevented it from fulfilling the request.

This, however, is not true for our exception. It's not an unexpected condition that prevent the server from fulfilling the request. It's a scheduled and wanted maintenance mode and the client can retry the very same request somewhen in the future again.